kratos/tests/auxiliar_files_for_python_unnitest/materials_files/materials_with_subproperties.json

```diff
@@ -0,0 +1,106 @@
+{
+    "properties" : [{
+        "model_part_name" : "Inlets",
+        "properties_id"   : 1,
+        "Material"        : {
+            "constitutive_law" : {
+                "name" : "HyperElastic3DLaw",
+                "name_after_merge_rule_of_mixtures" : "RuleOfMixturesLaw",
```

Comment:
I don't understand, why is this not right away the `RuleOfMixturesLaw`?

Reply:
I can remove, it is a reminder